user/views.py

```diff
@@ -0,0 +1,43 @@
+import json, bcrypt, jwt, re
+import my_settings
+
+
+from django.views import View
+from django.http import JsonResponse, HttpResponse
+from user.models import Member, RecentView
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            email = data.get('email')
+            password = data.get('password')
+            
+            if not validate_email(email):
+                raise ValidationError('INVALID_EMAIL')
```

Comment:
```suggestion
            if email:
               validate_email(email)
```
이렇게 바꾸는 이유는 email안에 정상적으로 키가 조회되서 데이터가 들어왔다면, 아래 함수를 실행시키면 메일에 문제가 없다면 
에러가 안날 것이고, 에러가 나면 ValidationError를 굳이 raise 안해줘도 exception이 발생하기 때문입니다.